src/gd_kotlin.cpp

```diff
@@ -4,6 +4,10 @@
 #include "core/os/os.h"
 #include "core/project_settings.h"
 #include "bootstrap.h"
+#include "type_manager.h"
+
+// If changed, remember to change also TransferContext::bufferCapacity on JVM side
+const int DEFAULT_SHARED_BUFFER_SIZE{20000000};
```

Comment:
I don't know if in cpp there is also another way of writing long numbers more readable like in kotlin with `_`. If so i would use that. Otherwise just mark as resolved ;-)

Reply:
IIRC it is `20'000'000`